sdk/test/Services/DynamoDBv2/UnitTests/Custom/DataModelOperationSpecificConfigTests.cs

```diff
@@ -18,5 +18,37 @@ public void BaseOperationConfig()
             // `ToDynamoDBOperationConfig` before updating this unit test
             Assert.AreEqual(6, typeof(BaseOperationConfig).GetProperties().Length);
         }
+
+        [TestMethod]
+        public void BatchGetConfig()
```

Comment:
non-blocking. but maybe worth adding unit test coverage for `ToDynamoDBOperationConfig`. Don't have to see this added, but just commenting since this is a new method, and these tests mainly seem to be a reminder to add new properties to `ToDynamoDbOperationConfig`

Reply:
Added in https://github.com/aws/aws-sdk-net/pull/3376/commits/68959295c0ea195936627845e99afdb54bc3fd91.

It's testing that one of the properties on the new config object is correctly overriding the context-level config by inspecting the low-level request.